configs/cornernet/cornernet_hourglass104_8xb6-210e-mstest_coco.py

```diff
@@ -139,3 +138,47 @@
 # USER SHOULD NOT CHANGE ITS VALUES.
 # base_batch_size = (8 GPUs) x (6 samples per GPU)
 auto_scale_lr = dict(base_batch_size=48)
+
+tta_model = dict(
```

Comment:
shoud create an individual tta cfg like other models.

Reply:
If you create separate files, file_client_args doesn't work very well, you need to change two places

Reply:
data_preprocessor is not processed well